app/pods/classroom/timeline/index/controller.js

```diff
@@ -44,6 +44,17 @@ export default class Overview extends Controller {
     return 0.84 * this.runAttempt.get('run.goodiesThreshold')
   }
 
+  @computed('course.topRun')
+  get getTotalContents() {
```

Comment:
don't prefix `get` to your computed properties. The idea is to be declarative so say "totalContents" intead of "getTotalContents"